bocas/artifacts/keras_history.py

```diff
@@ -1,16 +1,29 @@
-import pickle
-
 from bocas.artifacts.artifact import Artifact
+from .yaml_utils import parse_yaml_node
 
 
 class KerasHistory(Artifact):
     """KerasHistory is used to return the keras `model.fit()` history."""
 
+    yaml_tag = "!KerasHistory"
+
     def __init__(self, history, **kwargs):
         super().__init__(**kwargs)
-        self.history = history.history
+        try:
+            self.history = history.history
```

Comment:
Is this line needed for yaml parsing?  Maybe we can just save `history  = history`instead of `history = history.history`, and then we dont have to do this hack

Reply:
It was needed, but I agree this was not clean. To make this cleaner, I adjusted line 21. Since `.fit()` returns a history object and not just a dictionary we needed to adjust accordingly for yaml. The `to_yaml()` function now takes on this logic in a nicer way.

Reply:
looks like this works  - thanks!